temporal-sdk/src/main/java/io/temporal/worker/WorkerOptions.java

```diff
@@ -190,8 +193,17 @@ public Builder setLocalActivityWorkerOnly(boolean localActivityWorkerOnly) {
       return this;
     }
 
-    public Builder setDefaultDeadlockDetectionTimeout(long defaultDeadlockDetectionTimeout) {
-      this.defaultDeadlockDetectionTimeout = defaultDeadlockDetectionTimeout;
+    /**
+     * @param defaultDeadlockDetectionTimeoutMs time period in ms that will be used the detect workflows deadlock. Default is 1000ms. Default if 0.
```

Comment:
I would expand it a bit, for example:
"amount of time in milliseconds that workflow tasks are allowed to execute, if workflow task runs longer than specified interval it will fail automatically" 
Also "Default is 1000ms. Default if 0." sounds a little confusing. I don't think you need to mention "default if 0" part at all, but if you prefer to, I would just write something like "default is 1 sec, if set to 0 then default value is used."